src/Microsoft.ML.Core/Data/MetadataUtils.cs

```diff
@@ -367,6 +367,50 @@ public static bool IsHidden(this ISchema schema, int col)
             return !tmp || top != col;
         }
 
+        /// <summary>
+        /// Checks whether the schema columns match.
+        /// </summary>
+        /// <param name="schema">The schema</param>
+        /// <param name="otherSchema">The schema to compare against</param>
+        /// <returns>true if the schema columns match</returns>
+        public static bool EqualColumns(this ISchema schema, ISchema otherSchema)
```

Comment:
>EqualColumns [](start = 27, length = 12)

I don't know that this is a terribly common case. Maybe move this into the Parquet loader. If it turns out that lots of code uses stuff like this we can probably elevate it, but honestly I'm not sure this sort of thing happens that often.

Reply:
Fair enough.

---
In reply to: [201385299](https://github.com/dotnet/machinelearning/pull/472#discussion_r201385299) [](ancestors = 201385299)